src/content/guides/how-to-use-a-bridge/index.md

```diff
@@ -6,7 +6,7 @@ lang: en
 
 # How to bridge tokens to layer 2
 
-Layer 2s are scaling solutions for Ethereum which allow users to transact with increased speed and decreased costs. Many popular crypto projects are moving to layer 2s because of these benefits. The simplest way to move tokens from Ethereum to layer 2 is to use a bridge project.
+If everyone always uses Ethereum, it can get slow and expensive. One solution to this is to create new "layers": i.e. different networks which operate in similar ways to Ethereum itself. These so-called Layer 2s help reduce congestion and cost on Ethereum by processing many more transactions at lower fees, and only storing the result of these on Ethereum every so often. As such, these layers 2s enable us to transact with increased speed and decreased costs. Many popular crypto projects are moving to layer 2s because of these benefits. The simplest way to move tokens from Ethereum to layer 2 is to use a bridge.
```

Comment:
Just a few nitpicks on the opening line - "if everyone always uses Ethereum" is pretty vague - not *everyone* has to use Ethereum for gas fees to get high. Not sure what "always" is meant to convey - it doesn't matter how often users are transacting as long as there are enough transactions competing for block space to push up the cost of inclusion. And I guess "slow and expensive" is a bit of a weird one, because it's kind of one or the other - you can either accept a sufficiently high cost to ensure inclusion (expensive), or you can wait until gas drops to your cost threshold (slow).

```suggestion
If there is a lot of traffic on Ethereum, it can become expensive. One solution to this is to create new "layers": i.e. different networks which operate in similar ways to Ethereum itself. These so-called Layer 2s help reduce congestion and cost on Ethereum by processing many more transactions at lower fees, and only storing the result of these on Ethereum every so often. As such, these layers 2s enable us to transact with increased speed and decreased costs. Many popular crypto projects are moving to layer 2s because of these benefits. The simplest way to move tokens from Ethereum to layer 2 is to use a bridge.
```